From 5025c5cfd11045ebd489b9119aaa6e65353c38fe Mon Sep 17 00:00:00 2001
From: Nobuhiro Iwamatsu <nobuhiro.iwamatsu.x90@mail.toshiba>
Date: Thu, 26 Mar 2026 14:25:46 +0900
Subject: [PATCH 06097/12543] Revert "arm64: dts: renesas: r9a09g057: Add RTC
 node"

This reverts commit 09f116fe80c781d85c9cf8390aa4f3169dc0d8a8.

This reverts the CIP's commit, because tha same commit (d82a0f77999fbc2e1394a4265cc3332ad63ee137)
was applied in the LTS relelase.

Signed-off-by: Nobuhiro Iwamatsu <nobuhiro.iwamatsu.x90@mail.toshiba>
---
 arch/arm64/boot/dts/renesas/r9a09g057.dtsi | 15 ---------------
 1 file changed, 15 deletions(-)

diff --git a/arch/arm64/boot/dts/renesas/r9a09g057.dtsi b/arch/arm64/boot/dts/renesas/r9a09g057.dtsi
index 776ecbfbf07a..762dd6b7ab7d 100644
--- a/arch/arm64/boot/dts/renesas/r9a09g057.dtsi
+++ b/arch/arm64/boot/dts/renesas/r9a09g057.dtsi
@@ -601,21 +601,6 @@ rtc: rtc@11c00800 {
 			status = "disabled";
 		};
 
-		rtc: rtc@11c00800 {
-			compatible = "renesas,r9a09g057-rtca3", "renesas,rz-rtca3";
-			reg = <0 0x11c00800 0 0x400>;
-			interrupts = <GIC_SPI 524 IRQ_TYPE_EDGE_RISING>,
-				     <GIC_SPI 525 IRQ_TYPE_EDGE_RISING>,
-				     <GIC_SPI 526 IRQ_TYPE_EDGE_RISING>;
-			interrupt-names = "alarm", "period", "carry";
-			clocks = <&cpg CPG_MOD 0x53>, <&rtxin_clk>;
-			clock-names = "bus", "counter";
-			power-domains = <&cpg>;
-			resets = <&cpg 0x79>, <&cpg 0x7a>;
-			reset-names = "rtc", "rtest";
-			status = "disabled";
-		};
-
 		scif: serial@11c01400 {
 			compatible = "renesas,scif-r9a09g057";
 			reg = <0 0x11c01400 0 0x400>;
-- 
2.53.0

